servicebox-app/src/main/java/io/halkyon/model/ClaimingStatus.java

```diff
@@ -0,0 +1,10 @@
+package io.halkyon.model;
+
+import javax.persistence.Entity;
+
+import io.quarkus.hibernate.orm.panache.PanacheEntity;
+
+@Entity
+public class ClaimingStatus  extends PanacheEntity {
+    public int attempts;
```

Comment:
Shouldn't we have a initialized value to 1?

Reply:
Why? For me, it should be zero. However, the absence of this table means that were no retries made.

Reply:
ok, ok, or initialized to 0. I meant I was surprised to not have any value there

Reply:
It's now initialized to 0 in the Claim entity.